git-ftp

```diff
@@ -834,6 +878,9 @@ handle_action() {
 		pull)
 			action_pull
 			;;
+        clone)
```

Comment:
Please use tabs instead of spaces to be consistent with the rest of the project.